.github/workflows/rust-ci.yml

```diff
@@ -1,18 +1,16 @@
-# TODO: Replace this line with the commented ones to actually run the action in your repo(s)
-on: public
-# on:
-#   push:
-#     branches:
-#       - main
-#     tags:
-#       - "*"
-#   pull_request:
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
 
 name: CI
 jobs:
   lint:
     name: Lint
-    runs-on: ubuntu-latest
```

Comment:
You can keep ubuntu-latest now since its 20.04 today unless you specifically want to build with 20.04. :)

Reply:
I like using the explicit version since it will always do exactly what we want, even after they have added a later version that we  can switch to explicitly rather than implicitly like the previous change from 18 => 20.

Reply:
Makes sense :)